go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -139,12 +139,10 @@ func validateNetlinkState(nodeSubnet string) {
 
 	// Expect a route to the hybrid overlay CIDR via the .3 address
```

Comment:
@JacobTanenbaum comment is wrong now; not the .3 address anymore.

But also, we shouldn't be adding routes inside the pod netns anymore? eg netlink shouldn't care or have any of the .3 or whatever address these days. That should already be handled by the Reroute policy in `setupHybridLRPolicySharedGw()`.

Reply:
Eg, I don't think we need this bit after https://github.com/ovn-org/ovn-kubernetes/pull/3120 right?

Reply:
good spot, comment fixed